R/cov_ug.R

```diff
@@ -108,7 +147,7 @@ port <- function(N = 1, p = 3, d = 1, ug = NULL, zapzeros = TRUE) {
 #' igraph::print.igraph(ug)
 #' diagdom(ug = ug)
 #' @export
-diagdom <- function(N = 1, p = 3, d = 1, ug = NULL) {
+diagdom <- function(N = 1, p = 3, d = 1, ug = NULL, rfun = rnorm, ...) {
```

Comment:
This needs to be documented, otherwise travis build fails (I already fixed the error from travis, it could not find RBGL and graph from BioC)